### ISSUE TITLE
Project Still Active?

### ISSUE THREAD

**Issue by lonnietc** [castielblak/graphy-db#2]
Hello,

I have been searching for a simple C# Graph Database that I can use in a project and also learn from when I came across yours.

I am wondering if the project is still ative?

And also, I see the statement:

```
DbEngine engine = new DbEngine();
```

I am wondering where I define the database name to open and where the database files are stored?

Also, do you have some simple examples beyond what is given in the Remo as that example is a bit hard to follow.
Thanks and have a great day

**Comment by furesoft** [comment]
the database path can be configured by app config And here is a simple example: https://github.com/vkurenkov/graphy-db/blob/master/engine/GraphyDb/Program.cs
